nsot/models.py

```diff
@@ -808,8 +808,6 @@ def subnets(self, include_networks=True, include_ips=True, direct=False,
             broadcast_address__lte=self.broadcast_address
         )
 
-    # FIXME(jathan): This entire implementation needs to be revisited. It feels
-    # as if it's too complicated. It may not be, but it feels like it is.
     def get_next_network(self, prefix_length, num=None, as_objects=True):
```

Comment:
So looking at this it looks like you changed the default behavior to strict! I saw in #251 that you asked if we wanted the behavior to be a flag, and we do.

So we need "loose" allocation to be the default with "strict" as a flag. 

I apologize that I didn't get back to you sooner on this. I will keep a closer look now.